classpath/java/lang/Integer.java

```diff
@@ -139,4 +139,14 @@ public static Integer decode(String string) {
     }
     return new Integer(parseInt(string, 10));
   }
+
+  public static int numberOfLeadingZeros(int i) {
```

Comment:
It would be good to add a few test cases to the test suite to make sure this behaves correctly for e.g. negative numbers, very large positive numbers, etc.